src/components/VideoList/VideoList.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import videos from '../../mocks/youtube-videos-mock.json';
+import VideoCard from '../VideoCard';
+import { VideoWrapper, VideoFlex, SectionTitle } from './VideoList.styled';
+
+function VideoList() {
+  return (
+    <VideoWrapper>
+      <SectionTitle>Welcome!</SectionTitle>
+      <VideoFlex>
+        {videos.items.slice(1).map((video) => (
```

Comment:
Por qué ignoras el primer elemento?

Reply:
Porque noté que el primer elemento no tenía las misma keys que los demás y por lo tanto no podía asignarle el "videoId" como a los otros. ¿Hay alguna forma de asignarle un "key value" que no sea el index a elementos que no comparten las mismas propiedades?

Reply:
para evitar valores hardcodeados como en este caso, podrías usar un `filter`:
`videos.items.filter((video) => video.id.videoId).map((video) => (`
y así eliminas los videos que no tengan esa propiedad de tu Array
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
o crear un componente especial para los canales y hacer un rendereado condicional, pero eso sería más trabajo

si lo que quieres es un valor tipo fallback, puedes usar el operador `||` también:
`const a = video.id.videoId || video.snippet.title`
si el primer valor es falsy, toma el segundo